### PR TITLE
Create Miami page

### PR DESCRIPTION
**Pull request by ivan-kadykalo**
[DEMO LINK](https://ivan-kadykalo.github.io/layout_miami/)

### CODE REVIEW

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
https://www.loom.com/share/92aac1c2d2134c7b8f55b697c739e04c

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Where did you get your design?
There another images:
https://www.figma.com/file/nHz8bflIwJaWP3P99vKTH5/miami_home_new?node-id=16033%3A3

Also you don't have home block, so better leave # on footer logo link

**Comment by ivan-kadykalo** [comment]
> Where did you get your design? There another images: https://www.figma.com/file/nHz8bflIwJaWP3P99vKTH5/miami_home_new?node-id=16033%3A3
> 
> Also you don't have home block, so better leave # on footer logo link

All previous pictures are from forked repository.

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,425 @@
...
+            <a
+              href="#menu"
+              class="icon icon--menu">
+            </a>
+
+            <a href="#home" class="logo">
+              <img
+                src="images/logo.png"
+                alt="Miami Condo Kings logo"
+                class="logo__image">
```
Comment (AlinaMatuschak) [inline src/index.html:25]: Don't forget about style code

```suggestion
              <img
                src="images/logo.png"
                alt="Miami Condo Kings logo"
                class="logo__image"
              >
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,425 @@
...
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contact us</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 555-55-55"
+            class="menu__call menu__call--phone">
+            +1 234 555-55-55
+          </a>
```
Comment (AlinaMatuschak) [inline src/index.html:82]: Should fix that for all file

```suggestion
          <a
            href="tel:+1 234 555-55-55"
            class="menu__call menu__call--phone"
          >
            +1 234 555-55-55
          </a>
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,425 @@
...
+              </h2>
+            </div>
+
+            <div class="grid__item grid__item--7-12">
+              <a href="#" class="about__subtitle">Over 50 years experience in love.</a>
+              <p class="about__paragraph">Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+
+              <p class="about__paragraph">The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.Working directly with developers - so have more useful information about the profitable deals.</p>
+
+              <p class="about__paragraph about__paragraph--important">We will never be late!<br>Guaranteed secure transaction - 100%</p>
```
Comment (AlinaMatuschak) [inline src/index.html:115]: long lines

File: src/styles/blocks/footer.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.footer {
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  background-size: cover;
+  background-position: center;
+  height: 204px;
+  padding-bottom: 24px;
+
+  background-image: url(../images/newImg/img01.png);
```
Comment (AlinaMatuschak) [inline src/styles/blocks/footer.scss:10]:
```suggestion
  background: center / cover no-repeat url(../images/newImg/img01.png);
  height: 204px;
  padding-bottom: 24px;
```

File: src/styles/blocks/header.scss [hunk 5]
```diff
@@ -0,0 +1,68 @@
+.header {
+  background-size: cover;
+  background-position: center;
+  background-image: url(../images/newImg/img02.png);
```
Comment (AlinaMatuschak) [inline src/styles/blocks/header.scss:4]: use background too

File: src/styles/blocks/icon.scss [hunk 6]
```diff
@@ -0,0 +1,23 @@
+.icon {
+  display: block;
+  width: 100%;
+  padding-bottom: 100%;
+
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+  // position: relative;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/icon.scss:9]: delete comment

File: src/styles/blocks/page.scss [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    &::after {
+      content: "";
+      display: block;
+      height: 16px;
+      width: 23px;
+      margin: 4px auto 0;
+      background-image: url(../images/crown.svg);
+      background-position: center;
+      background-size: contain;
+      background-repeat: no-repeat;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/page.scss:63]: 😉

**Review by AlinaMatuschak — APPROVED** [review]
Good work 💪

**Pull request closed without merging**